src/lib/bootloader/systeminfo.rb

```diff
@@ -137,6 +138,50 @@ def s390_secure_boot_active?
         false
       end
 
+      # Return secure boot status on ppc
+      #
+      # nil - no support
+      # 0   - disabled
+      # 1   - enabled in audit-only mode
+      # 2+  - enabled in enforcing mode
+      def ppc_secure_boot
```

Comment:
all ppc specific methods should have something like
```
      def ppc_secure_boot
          return nil unless Yast::Arch.ppc
```

so it does not affect non ppc code.

Reply:
It reads a ppc-specific file so this is pretty safe.

Reply:
yeap, but it do some unnecessary file read and also makes code more obvious, that it is ppc specific. as ibm,secure-boot can be e.g. added to s390x which can do some side effects.

Reply:
s390 has /sys/firmware/ipl/secure

Reply:
it is current state, but show know what brings future? So I prefer more exact code that survive potential changes on other architectures.